tekton/cronjobs/robocat-triggers-deploy-latest-cron/cronjob.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: tekton-release-cd-trigger
+spec:
+  schedule: "0 3 * * *"
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: trigger
```

Comment:
ohhhhhh https://github.com/tektoncd/plumbing/blob/master/tekton/cronjobs/robocat-pipeline-deploy-latest-cron/cronjob.yaml has a container named "trigger" too - i didnt look closely enough and I thought that https://github.com/tektoncd/plumbing/blob/master/tekton/cronjobs/robocat-pipeline-deploy-latest-cron/cronjob.yaml was deploying triggers 🤦‍♀  i get it now 😅

Reply:
Yeah, I guess the name `trigger` there can be confusing. Sorry 🙏

Reply:
np!